tools/templates/iiot_deployment_linux.yml

```diff
@@ -56,3 +56,6 @@ jobs:
         az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/$(setVersionInfo.Version_Prefix)/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment
         az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment.pdb --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/$(setVersionInfo.Version_Prefix)/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment.pdb
         az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment.md5 --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/$(setVersionInfo.Version_Prefix)/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment.md5
+        az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/latest/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment
+        az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment.pdb --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/latest/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment.pdb
```

Comment:
why are we publishing the pdb files? for debugging purposes?

Reply:
Yes, that's the idea.